README.md

```diff
@@ -1,5 +1,5 @@
 
-[![Build Status](https://travis-ci.org/flexion/hud-disaster-data.svg?branch=master)](https://travis-ci.org/flexion/hud-disaster-data)
+Staging [![Build Status](https://travis-ci.org/flexion/hud-disaster-data.svg?branch=master)](https://travis-ci.org/flexion/hud-disaster-data)
```

Comment:
I think it might be good to see both, like this...

[![Build Status](https://img.shields.io/travis/flexion/hud-disaster-data/master.svg?label=build-staging)](https://img.shields.io/travis/flexion/hud-disaster-data/master.svg)
[![Build Status](https://img.shields.io/travis/18f/hud-disaster-data/master.svg?label=build-prod)](https://img.shields.io/travis/18f/hud-disaster-data/master.svg)